PendingReleaseNotes

```diff
@@ -89,6 +89,9 @@
   notifications needs to pull them (instead of the notifications be pushed 
   to it), an external message bus (e.g. rabbitmq, Kafka) should be used for 
   that purpose.
+* RGW: during upgrades, if newer version RGWs will be creating topic and notification
+  configuration, the older version RGWs will ignore them. Any existing topic and
+  notification configuration would be used by the newer version RGWs.
```

Comment:
> older version RGWs will ignore them

i don't think that's true. if an old rgw tries to read a new topic, it'll get a decode error. that should cause the whole s3 op to fail, effectively making the bucket unwritable on old gateways

> The serialized format of notification topics has changed, so that new/updated topics will be unreadable by old RGWs. We recommend completing the RGW upgrades before creating or modifying any notification topics.

Reply:
thanks, lgtm!